doc/architecture/platforms/index.rst

```diff
@@ -0,0 +1,15 @@
+Platforms
+===================
+
+CCF can run on several platforms, which platform CCF runs on will have impacts on how attestations are verified and therefore what security guarantees you have.
+
+.. toctree::
+    :maxdepth: 1
+
+    sgx
+    snp
+    virtual
+
+Which platform code is run is determined partly by the compile time flag ``COMPILE_TARGETS`` and partly by runtime detection of the hardware.
+
+CCF networks running a mix of node platforms are not currently supported.
```

Comment:
```suggestion
.. note:: CCF networks running a mix of node platforms are not currently supported.
```

Renders nicely in the docs.

Reply:
https://github.com/microsoft/CCF/pull/4330/commits/43e51acc4b4a0aee98d8e892ffb419bf21840779